### PR TITLE
KFSPTS-32438 update req notes tag to handle LocalDateTime correctly

### PR DESCRIPTION
**Pull request by jhulslander**

### CODE REVIEW

**Review by cah292 — COMMENTED** [review]
There's one minor formatting/naming spot that would be good to clean up, but otherwise the changes look good. If you think it's not worth cleaning it up at this time, please let me know so that I (or someone else) can proceed with merging.

File: src/main/java/edu/cornell/kfs/sys/web/CuWebUtilities.java [hunk 1]
```diff
@@ -99,5 +101,10 @@ private static String updateLookupParameter(String lookupParameter) {
         }
         return StringUtils.join(formFieldName, CUKFSConstants.COLON, personFieldName);
     }
+    
+    public static long convertLocalDateTimeToMilliseconds(LocalDateTime localdateTime) {
```
[the remark this answers is not in the document]
Comment (jhulslander) [reply src/main/java/edu/cornell/kfs/sys/web/CuWebUtilities.java:105]: will fix

**Comment by jhulslander** [comment]
this is ready for review again

**Review by cah292 — APPROVED** [review]
Changes look good, will merge once the unit tests pass.

**Pull request merged**